src/azure-cli/azure/cli/command_modules/acr/commands.py

```diff
@@ -233,3 +257,21 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
 
     with self.command_group('acr', acr_check_health_util) as g:
         g.command('check-health', 'acr_check_health')
+
+    with self.command_group('acr scope-map', acr_scope_map_util, is_preview=True) as g:
+        g.command('create', 'acr_scope_map_create')
+        g.command('delete', 'acr_scope_map_delete')
+        g.command('update', 'acr_scope_map_update')
+        g.command('show', 'acr_scope_map_show')
+        g.command('list', 'acr_scope_map_list')
+
+    with self.command_group('acr token', acr_token_util, is_preview=True) as g:
+        g.command('create', 'acr_token_create')
+        g.command('delete', 'acr_token_delete')
+        g.command('update', 'acr_token_update')
+        g.command('show', 'acr_token_show')
+        g.command('list', 'acr_token_list')
+        g.command('credential delete', 'acr_token_credential_delete')
```

Comment:
Why is this not under `az acr token credential`?

Reply:
As discussed offline, it needs a different tokens client for credential deletion.
Credential generation uses registries client.